### PR TITLE
EPMDJ-9927  Cookies not set for Firefox tests

### PR DESCRIPTION
**Pull request by Mazel-Tovr**

### CODE REVIEW

File: src/runtimeMain/kotlin/com/epam/drill/test/agent/instrumentation/http/selenium/Selenium.kt [hunk 1]
```diff
@@ -28,13 +27,18 @@ const val EXTENSION_NAME = "header-transmitter.xpi"
...
-    private val Command = "org.openqa.selenium.remote.Command"
-    private val ImmutableMap = "com.google.common.collect.ImmutableMap"
-    private val ImmutableList = "com.google.common.collect.ImmutableList"
-    private val Cookie = "org.openqa.selenium.Cookie"
-    private val DesiredCapabilities = "org.openqa.selenium.remote.DesiredCapabilities"
-    private val Proxy = "org.openqa.selenium.Proxy"
-    private val initPages = """"about:blank", "data:,""""
+
+    companion object {
+        const val addDrillCookiesMethod = """addDrillCookies"""
```
Comment (LikhanovMaxim) [inline src/runtimeMain/kotlin/com/epam/drill/test/agent/instrumentation/http/selenium/Selenium.kt:32]:
```suggestion
        const val addDrillCookiesMethod = "addDrillCookies"
```

File: src/runtimeMain/kotlin/com/epam/drill/test/agent/instrumentation/http/selenium/Selenium.kt [hunk 2]
```diff
@@ -28,13 +27,18 @@ const val EXTENSION_NAME = "header-transmitter.xpi"
...
+        private const val Command = "org.openqa.selenium.remote.Command"
+        private const val ImmutableMap = "com.google.common.collect.ImmutableMap"
+        private const val ImmutableList = "com.google.common.collect.ImmutableList"
+        private const val Cookie = "org.openqa.selenium.Cookie"
+        private const val DesiredCapabilities = "org.openqa.selenium.remote.DesiredCapabilities"
+        private const val Proxy = "org.openqa.selenium.Proxy"
+        private const val initPages = """"about:blank", "data:,""""
+        private const val isFirefoxDriver = """this instanceof org.openqa.selenium.firefox.FirefoxDriver"""
```
Comment (LikhanovMaxim) [inline src/runtimeMain/kotlin/com/epam/drill/test/agent/instrumentation/http/selenium/Selenium.kt:40]:
```suggestion
        private const val initPages = "\"about:blank
        ", "data:,\""
        private const val isFirefoxDriver = "this instanceof org.openqa.selenium.firefox.FirefoxDriver"
```

File: src/runtimeMain/kotlin/com/epam/drill/test/agent/instrumentation/http/selenium/Selenium.kt [hunk 3]
```diff
@@ -95,8 +99,8 @@ class Selenium : TransformStrategy() {
         ctClass.addMethod(
             CtMethod.make(
                 """
-                    public void addDrillCookies() {
-                        if ($IF_CONDITION && !$IS_HEADER_ADDED){
+                    public void $addDrillCookiesMethod() {
+                        if ($isFirefoxDriver && $IF_CONDITION) {
```
Comment (LikhanovMaxim) [inline src/runtimeMain/kotlin/com/epam/drill/test/agent/instrumentation/http/selenium/Selenium.kt:103]: rename IF_CONDITION

**Review by LikhanovMaxim — CHANGES_REQUESTED** [review]

File: build.gradle.kts [hunk 4]
```diff
@@ -116,7 +116,6 @@ kotlin {
             dependsOn(commonMain)
             dependencies {
                 implementation("com.epam.drill:jvmapi:$drillJvmApiLibVersion")
-                implementation("com.epam.drill.interceptor:http:$drillHttpInterceptorVersion")
```
Comment (Mazel-Tovr) [inline build.gradle.kts:119]: you forgot  to remove version from gradle.properties

**Pull request merged**